src/monitor/node_active_protocol.c

```diff
@@ -261,14 +262,21 @@ node_active(PG_FUNCTION_ARGS)
 
 	checkPgAutoFailoverVersion();
 
+	ereport(INFO, (errmsg("node_active is called with %s, %d, " UINT64_FORMAT ,
+						   nodeName, nodePort,  currentLSN)));
+
+
```

Comment:
This looks like debug level logs, do we need them once the PR is merged?

Reply:
it could be removed